src/interactions.js

```diff
@@ -28,26 +28,31 @@ module.exports = function(component, renderer, container) {
 	}
     });
 
-    container.addEventListener("mouseup", (e) => {
+    window.addEventListener("mouseup", (e) => {
```

Comment:
Won't this make it so that whenever you click and drag on the browser window, it will rotate the molecule? Is it possible to see whether the mouse is over the canvas before applying this event handler?

Reply:
No it won't, one must click on the molecule element in order to engage dragging and then my changes simply allow the mouse to continue to drag (or release) the molecule even if while dragging the mouse cursor leaves the molecule element. There is already code in place which checks to see if drag is engaged before executing whats in the `mouseup` and `mousemove` event handlers.